tests/integration/test_actor_api_helpers.py

```diff
@@ -330,21 +329,26 @@ async def main() -> None:
             async with Actor:
                 print('Starting...')
                 input = await Actor.get_input() or {}
-                counter_key = cast(str, input.get('counter_key'))
-                cnt = await Actor.get_value(counter_key, 0)
+                counter_key = input.get('counter_key')
+                cnt = await Actor.get_value(counter_key, 0)  # type: ignore
```

Comment:
Is it actually necessary to pass the `counter_key` through the input? You could get rid of the type violations if you just hardcoded the `'reboot_counter'` here, it's fine

Reply:
Sure, no problem